internal/resource/cuda-device.go

```diff
@@ -96,3 +96,5 @@ func (d *cudaDevice) IsMigCapable() (bool, error) {
 func (d *cudaDevice) IsMigEnabled() (bool, error) {
 	return false, nil
 }
+
+func (d *cudaDevice) GetDisplayMode() (string, error) { return "unknown", nil }
```

Comment:
nit: let's have this span multiple lines.

Reply:
Fixed.